release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -538,9 +538,44 @@ Starting in {product-title} 4.14, Extended Update Support (EUS) is extended to t
+The Insights Operator now collects more {product-title} container log data from namespaces prefixed with either the  “openshift-” or “kube-” prefix and generates recommendations much faster.
+Enhancements have also been made to give you more flexibility in how the data to be collected gets defined for your service.
+
+==== Rapid Recommendations
+
+This release introduces a new feature called Rapid Recommendations, which provides a more dynamic and version-independent mechanism for remotely configuring the rules that determine which data the Insights Operator collects.
+
+Rapid Recommendations builds on the existing conditional data gathering mechanism.
+The Insights Operator connects to a secure remote endpoint service running on `/console.redhat.com` to retrieve definitions that contain the rules for determining which container log messages are filtered and collected by Red Hat.
+
+The conditional data-gathering definitions, also referred to as rules, get configured through an attribute named `conditionalGathererEndpoint` in the link:https://github.com/openshift/insights-operator/blob/master/config/pod.yaml[`pod.yml`] configuration file.
+
+
+```bash
+conditionalGathererEndpoint: https://console.redhat.com/api/gathering/v2/%s/gathering_rules
+```
+
+NOTE: Previously, the rules for determining the data that the Insights Operator collects were hard-coded and tied to the corresponding {product-title} version.
+The preconfigured endpoint URL now provides a placeholder (`%s`) for defining a target version of {product-title}.
+
+==== More data collected and recommendations added
+
+The Insights Operator now gathers more data to detect the following scenarios, which other applications can use to generate remedial recommendations to proactively manage your {product-title} deployments:
+
+// Engineering reference: CCXDEV-12899
+* Detects pods and namespaces that use the link:https://access.redhat.com/articles/7065170[deprecated OpenShiftSDN CNI plugin] and generates a recommendation for the possible actions you should take depending on the data collected from your deployment.
+// Engineering reference: OSPRH-5904
+* Collects custom resource definitions (CRD) from {rh-openstack}.
+// Engineering reference: CCXDEV-13001
+* Collects the `haproxy_exporter_server_threshold` metric to detect the problem and remediation reported in link:https://issues.redhat.com/browse/OCPBUGS-36687[*OCPBUGS-36687*].
+// Engineering reference: CCXDEV-12758
+* Collects data to detect custom Prometheus Alertmanager instances that are not in the "openshift-monitoring" namespace because they could potentially impact the management of corresponding resources.
```

Comment:
```suggestion
* Collects data to detect custom Prometheus Alertmanager instances that are not in the `openshift-monitoring` namespace because they could potentially impact the management of corresponding resources.
```